certify-default.properties

```diff
@@ -2,6 +2,18 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
+### DataProvider plugin specific changes
+# tells Certify to become a proxy to other issuers via VCIssuancePlugin impl
+mosip.certify.issuer=PluginIssuer
+mosip.certify.data-provider.identifier.column=id
+mosip.certify.data-provider.fields.include=id,name,phoneNumber,dateOfBirth,highestEducation,typeOfHouse,numberOfDependents,works,landArea,landOwnershipType,primaryCropType,secondaryCropType,maritalStatus
+mosip.certify.plugin.csv.file.uri=https://raw.githubusercontent.com/jainhitesh9998/digital-credential-plugins/refs/heads/develop/mock-certify-plugin/src/test/resources/farmer_identity_data.csv
```

Comment:
All properties introduced in plugin code, should never be in the certify-default.properties, it should be present only in the respective plugin specific usecase property file.

Kind follow the sections like below with empty value to call out the properties part of the certify code, but it is expected to be overridden in the plugin-usecase property files. One example is mosip.certify.issuer

https://github.com/mosip/inji-config/blob/v0.2.0/certify-default.properties#L141

Reply:
overridden values moved to the overridden section of default properties.